session/session.go

```diff
@@ -1159,10 +1159,66 @@ func (s *session) PrepareStmt(sql string) (stmtID uint32, paramCount int, fields
 	return prepareExec.ID, prepareExec.ParamCount, prepareExec.Fields, nil
 }
 
+// CachedPlanExec short path ONLY for cached "point select plan" execution
+func (s *session) CachedPlanExec(ctx context.Context,
```

Comment:
take care some trace / metric  is missed in this new short-cut

Reply:
thx, PTAL